socorro/unittest/external/es/test_correlations.py

```diff
@@ -53,3 +124,90 @@ def test_create_correlations_index(self):
 
         # We should be able to create that index again without any errors.
         correlations.create_correlations_index(es_index)
+
+
+class TestCoreCounts(BaseTestCorrelations):
+
+    def test_store(self):
+        correlations = CoreCounts(config=self.config)
+        today = datetime.datetime.utcnow()
+        correlations.store(
+            sample_core_counts_summary,
+            today.strftime('%Y%m%d'),
+            'core-counts',
+            'Firefox_43.0.1'
+        )
+        es_index = correlations.get_index_for_date(today.date())
+        correlations.create_correlations_index(es_index)
+        self.indices_for_deletion.append(es_index)
```

Comment:
Unless I missed it, you do not set a special value for this field in unit tests. You should, otherwise you risk deleting an actual index in your local database. Or mine. Maybe that's not the most terrible thing that can happen, but it's still annoying to see tests delete your data. :)

You can set a different default for tests here: https://github.com/peterbe/socorro/blob/bug-1249359-store-correlations-in-es/socorro/unittest/external/es/base.py#L22